src/components/fundamentals/status-dot/index.tsx

```diff
@@ -1,9 +1,9 @@
-import React from "react"
 import clsx from "clsx"
+import React from "react"
 
 type StatusDotProps = {
   title: string
-  variant: "primary" | "danger" | "warning" | "success"
+  variant: "primary" | "danger" | "warning" | "success" | "default"
 } & React.HTMLAttributes<HTMLDivElement>
 
 const StatusDot: React.FC<StatusDotProps> = ({
```

Comment:
When you merge `feat/revamp` this will have changed name to <StatusIndicator>, and you will need to update the import a few places that I have marked 😄